bucket/cacert.json

```diff
@@ -7,10 +7,13 @@
     "hash": "cb2eca3fbfa232c9e3874e3852d43b33589f27face98eef10242a853d83a437a",
     "checkver": {
         "url": "https://curl.haxx.se/docs/caextract.html",
-        "re": "cacert-([\\d-]{10}).pem"
+        "regex": "cacert-([\\d-]+)\\.pem"
     },
     "post_install": "if (Test-Path \"$(appdir curl $global)\") {cp \"$dir\\cacert.pem\" \"$(appdir curl $global)\\current\\bin\\curl-ca-bundle.crt\"}",
     "autoupdate": {
-        "url": "https://curl.haxx.se/ca/cacert-$version.pem#/cacert.pem"
+        "url": "https://curl.haxx.se/ca/cacert.pem",
```

Comment:
I do not think that changing version specific URL into static one is worth hash extraction.